templates/advantage/table/_attached-machines.html

```diff
@@ -3,32 +3,49 @@
   <div class="row">
     <div class="col-start-large-2 col-10">
       <div class="row">
-          <h5>Welcome to your new subscription</h5>
-          <p class="u-no-margin--bottom">If you’re using Ubuntu 14.04 ESM or 20.04 LTS, first attach each machine.</p>
+        <div class="advantage-table-section">
+            <h5>Welcome to your new subscription</h5>
+            <p class="u-no-margin--bottom">If you’re using Ubuntu 16.04 or 18.04 LTS you may need to install the UA client first.</p>
+            <div class="advantage-table-instruction">
+              <span>Add the PPA</span>
+              <code>sudo add-apt-repository ppa:ua-client/stable<br/>
+              sudo apt-get update</code>
+              <hr />
+              <span>Install the UA client</span>
+            <code>sudo apt install ubuntu-advantage-tools</code>
+            </div>
+          </div>
+        <div class="advantage-table-section">
+          <p class="u-no-margin--bottom">Once installed or if you’re using Ubuntu 14.04 ESM or 20.04 LTS, first attach each machine.</p>
           <div class="advantage-table-instruction">
             <span>To attach a machine</span>
             <code>sudo ua attach {{ contract['token'] }}</code>
             <hr />
             <span>To detach a machine</span>
             <code>sudo ua detach</code>
           </div>
-          <p class="u-hide--small">
-            Once a machine is attached, you can configure its services.&nbsp;&nbsp;
-            <button class="p-button is-dense u-no-margin--bottom js-entitlements-callout" data-entitlements='{{ contract["entitlements"]|tojson }}'>Where?</button>
-          </p>
-          <p>If you’re using Ubuntu 16.04 or 18.04 LTS, you don’t need to attach machines.</p>
+        </div>
       </div>
     </div>
   </div>
-  <script src="{{ versioned_static('js/dist/ua-entitlements-callout.js') }}" defer></script>
 </td>
 {% else %}
 <td id="expanded-row-token-{{ outer_loop.index }}-{{ loop.index }}" class="p-table-expanding__panel" aria-hidden="true">
   <div class="row">
     <div class="col-start-large-2 col-10">
-      <div class="row">
         <div class="advantage-table-section">
-          <h5>Ubuntu 14.04 ESM and 20.04 LTS</h5>
+            <p class="u-no-margin--bottom">If you’re using Ubuntu 16.04 or 18.04 LTS you may need to install the UA client first.</p>
+            <div class="advantage-table-instruction">
+              <span>Add the PPA</span>
+              <code>sudo add-apt-repository ppa:ua-client/stable<br/>
+              sudo apt-get update</code>
+              <hr />
+              <span>Install the UA client</span>
+            <code>sudo apt install ubuntu-advantage-tools</code>
+            </div>
+          </div>
+        <div class="advantage-table-section">
+          <p class="u-no-margin--bottom">Once installed or if you’re using Ubuntu 14.04 ESM or 20.04 LTS, first attach each machine.</p>
```

Comment:
we also need to update the ESM screen (it's now 14.04 or 16.04, not just 14.04)
Similarly, Livepach can get activated via the ua client for each Ubuntu LTS now. We don't need separate "Ubuntu 16.04 LTS and 18.04 LTS" instructions anymore